_methods/recruiting.md

```diff
@@ -24,10 +24,10 @@ timeRequired: 1–2 weeks for 5–10 participants
 - The new or existing website.
 - Existing mailing lists.
 
-{% include header-gov-research.html %}
+<section class="method--section method--section--government-considerations" markdown="1" > {% include header-gov-research.html %}
 
 No PRA implications. No information is collected from members of the public.
-
+</section>
 <!-- nonPrintablecontent:
```

Comment:
@andrewmaier  do you know what's up with this commented out code?

Reply:
It used to link to a ConsentForm which was under redesign. It should now link to https://methods.18f.gov/participant-agreement, if anything!